sphinx/search/__init__.py

```diff
@@ -495,7 +495,7 @@ def _visit_nodes(node):
                     nodetext = re.sub(r'<[^<]+?>', '', nodetext)
                     word_store.words.extend(split(nodetext))
                 return
-            elif (isinstance(node, nodes.meta)  # type: ignore[attr-defined]
```

Comment:
Does the elif fit on a single line now?

Reply:
no idea. Sphinx is the only python codebase i've ever interacted with that doesn't use a formatter. I've got better things to do than manually tweak python formatting